src/python/pants/backend/docker/goals/package_image_test.py

```diff
@@ -533,3 +539,40 @@ def check_docker_proc(process: Process):
         Address("docker/test", target_name="img1"),
         process_assertions=check_docker_proc,
     )
+
+
+def test_docker_build_secrets_option(rule_runner: RuleRunner) -> None:
+    rule_runner.write_files(
+        {
+            "docker/test/BUILD": dedent(
+                """\
+                docker_image(
+                  name="img1",
+                  secrets={
+                    "mysecret": "/var/run/secrets/mysecret",
+                    "password": "password",
```

Comment:
As discussed in DM, I was really confused that `password` is a file path rather than string. I recommend calling it something like `local_password_file`